common/src/lib.rs

```diff
@@ -56,9 +56,9 @@ mod ibeji_common_tests {
     fn find_full_path_test() {
         set_dtdl_path();
 
-        let find_full_path_result = find_full_path("samples/remotely_accessible_resource.json");
```

Comment:
should this filename be a constant?

Reply:
This is a test.  No need for this to be a constant.

Reply:
the filename is repeated multiple times here though, which is why I suggest making it a constant local to this test